plugins/manipulator.cpp

```diff
@@ -2306,6 +2319,20 @@ DFhackCExport command_result plugin_init ( color_ostream &out, vector <PluginCom
     return CR_OK;
 }
 
+DFhackCExport command_result plugin_onstatechange(color_ostream &out, state_change_event event)
+{
+    switch (event) {
+    case SC_MAP_LOADED:
+        if (is_enabled)
+            enable_alchemist(out);
```

Comment:
I feel like this should be done by autohauler instead. Not everyone who uses manipulator necessarily uses autohauler, and this will affect other things (e.g. `tweak block-labors`).

Reply:
done